mix.exs

```diff
@@ -45,8 +45,7 @@ defmodule Philomena.MixProject do
       {:plug_cowboy, "~> 2.3"},
       {:phoenix_slime, "~> 0.13"},
       {:ecto_network, "~> 1.3"},
-      {:pow, "~> 1.0"},
-      {:bcrypt_elixir, "~> 2.2"},
+      {:bcrypt_elixir, "~> 2.0"},
```

Comment:
please change this back to bcrypt_elixir ~> 2.2

Reply:
done